integrated_channels/blackboard/models.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+Database models for Enterprise Integrated Channel Blackboard.
+"""
+
+from simple_history.models import HistoricalRecords
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from integrated_channels.integrated_channel.models import EnterpriseCustomerPluginConfiguration
+from .transmitters.content_metadata import BlackboardContentMetadataTransmitter
+from .exporters.content_metadata import BlackboardContentMetadataExporter
+
+
+@python_2_unicode_compatible
+class BlackboardEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguration):
+    """
+    The Enterprise-specific configuration we need for integrating with Blackboard.
+
+    .. no_pii:
+    """
+
+    client_id = models.CharField(
+        max_length=255,
+        null=True,
+        verbose_name="API Client ID",
+        help_text=(
+            "The API Client ID provided to edX by the enterprise customer to be used to make API "
+            "calls on behalf of the customer."
+        )
+    )
+
+    client_secret = models.CharField(
```

Comment:
Something I always like to think about when storing secrets is how they're being stored, and how they're being displayed. I don't know what the considerations are here, but, food for thought.

Reply:
Good thought ! we should probably talk about this in the team channel. Generally all channels seem to be storing this as plain text probably because you need this plus a bunch of other info to get a valid token (client_id, redirect_uri all of which have to match).